WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2322,3 +2322,19 @@ PrefOOOExternalMsg          = margin-top:@PrefViewGutterSpace@;
 PrefOOOTimePeriodCell       = padding:@PrefSectionInternalSpacing@ 5px;
 PrefOOOMiniDropDown         = @PrefRowMiniSelectWidget@;
 PrefOOODurationLabel        = text-transform:lowercase;
+
+######################################
+#   PREFERENCES -Mail Page
+######################################
+
+PrefMailComposeRowValueWrapperWidth            = 22%
+PrefMailBlockMessageCellWidth                  = 55%
+PrefMailComposeRowSelectWidgetWidth            = 130px
+PrefMailComposeRowColorPickerWidgetWidth       = 1%
+PrefMailComposeLabel                           = padding-top:@SkinWrapperPadding@
+PrefMailComposeRowLabel                        = padding:11px @SkinViewInnerSpacing@; vertical-align:top
```

Comment:
Instead of `11px` value, you can use `PrefViewGutterSpace` variable, if that looks good.

Also instead of `SkinViewInnerSpacing` you can use `PrefViewInnerSpacing`. It's having same value but we redeclare it for preference section, so that we can override it if needed.

https://github.com/Zimbra/zm-web-client/blob/b6c91ee06f02822118b731840c22bc166d26796e/WebRoot/skins/_base/base4/skin.properties#L2249